app/styles/side-bar.less

```diff
@@ -50,7 +50,10 @@
 @media (min-width: @screen-sm-min) {
   #app-sidebar {
     width: @sidebar-width;
-    min-height: 600px;
```

Comment:
Deze minimumhoogte is niet nodig als je gebruik maakt van het after-element die dient als decorator element om de doorlopende achtergrond te faken.